constants/index.ts

```diff
@@ -46,6 +46,7 @@ export const RESPONSE_MESSAGES = {
   FAILED_TO_FETCH_PRICE_FROM_API_500: (day: string, ticker: string) => { return { statusCode: 500, message: `Failed to fetch ${ticker} price for day ${day}` } },
   MISSING_WS_AUTH_KEY_400: { statusCode: 400, message: 'Missing WS_AUTH_KEY environment variable' },
   MISSING_PRICE_FOR_TRANSACTION_400: { statusCode: 400, message: 'Missing price for transaction.' },
+  INVALID_PRICES_FOR_TX_ON_CSV_CREATION_500: (pricesLenght: number) => {  return  {statusCode: 500, message: `Missing price for transaction in CSV creation. ${pricesLenght}` }},
```

Comment:
The variable name and message are not right:

To be more precise, is the number of prices that is wrong, not the prices themselves. But more than that, this happens if there is a number != 1 of prices, so there could be missing prices or there could be more prices than expected. This error says "Missing price", which could not always be the case, and just throw a random number at the end without explain what it is. A better approach here would be:
```
INVALID_PRICES_AMOUNT_FOR_TX_ON_CSV_CREATION_500: (pricesLenght: number) => { return  {
statusCode: 500, 
message: `Wrong number of prices for transactions group in CSV creation. Expected 1, got ${pricesLenght}`. 
}},
```